src/VirtualMonoRepo/InitializeVMR.proj

```diff
@@ -0,0 +1,122 @@
+    <TmpDir Condition=" '$(TmpDir)' == '' ">$(RepoRoot)tmp/</TmpDir>
+    <VmrDefaultBranch Condition=" '$(VmrDefaultBranch)' == '' ">main</VmrDefaultBranch>
+
+    <!-- We set this for the source build targets to work well -->
+    <TarballDir>$(VmrDir)</TarballDir>
+    <InitializeVMR>true</InitializeVMR>
+  </PropertyGroup>
+
+  <Import Project="../SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets" />
+
+  <PropertyGroup>
+    <VmrGitInfoDir>$(TarballGitInfoDir)</VmrGitInfoDir>
+  </PropertyGroup>
+
+  <UsingTask TaskName="Microsoft.DotNet.VirtualMonoRepo.Tasks.VirtualMonoRepo_Initialize" AssemblyFile="$(VirtualMonoRepoTasksAssembly)" />
+
+  <Target Name="Build" DependsOnTargets="InitializeVMR" />
+
+  <!--
+    This target initializes a fresh VMR from scratch by assembling it similarly to the tarball.
+    Instead of cloning the repositories, we pull the sources via the `darc vmr initialize` command.
+    Most of the targets are reused from the tarball generation process (SourceBuildArcadeTarball.targets).
+  -->
+  <Target Name="InitializeVMR" DependsOnTargets="GenerateFullNuGetVersion;
+                                                 GenerateVersionFile;
+                                                 InitializeCleanVmr;
+                                                 CopyTarballContent;
+                                                 CommitInitialContent;
+                                                 SetupSelfGithubInfo;
+                                                 InitializeRepoAndDependentsRecursive;
+                                                 CommitGitInfoFiles;
+                                                 CopyTextOnlyPackages;
+                                                 CommitTextOnlyPackages">
+    <Message Text="VMR was successfully initialized in '$(VmrDir)'" Importance="High" />
+  </Target>
+
+  <Target Name="InitializeCleanVmr">
+    <Error Text="VmrDir not specified" Condition=" '$(VmrDir)' == '' " />
+
+    <RemoveDir Directories='$(VmrDir)' Condition=" EXISTS('$(VmrDir)') " />
+    <MakeDir Directories="$(VmrDir)" />
+    <MakeDir Directories="$(VmrGitInfoDir)" Condition=" '$(VmrGitInfoDir)' != '' " />
+    <MakeDir Directories="$(TmpDir)" Condition=" !EXISTS('$(TmpDir)') " />
+
+    <Exec Command="git init -b $(VmrDefaultBranch)" WorkingDirectory="$(VmrDir)" />
+  </Target>
+
+  <Target Name="CommitInitialContent">
+    <Exec Command="git add -A" WorkingDirectory="$(VmrDir)" />
+    <Exec Command="git commit -m 'VMR initialized'" WorkingDirectory="$(VmrDir)" />
+  </Target>
+
+  <Target Name="InitializeRepoAndDependentsRecursive"
+    DependsOnTargets="GetSourceBuildIntermediateNupkgNameConvention">
+
+    <Message Text="--> Initializing repo $(SourceBuildRepoName)" Importance="High" />
+    <PropertyGroup>
+      <SourceDir>$(SourceBuildRepoName)/</SourceDir>
+      <TarballRepoSourceDir>$(TarballSourceDir)$(SourceDir)</TarballRepoSourceDir>
```

Comment:
The use of `Tarball` in property names here is confusing in the context of the vmr.

Reply:
Removed all `Tarball*` properties from this file